samples/matter/light-bulb/readme.rst

```diff
@@ -0,0 +1,150 @@
+#. You shuold see now fast blinking Blue led when BLE connection is established.
+#. ``Aplle Home`` will warning Uncertificated Acessory so just click ``Add Anyway`` for continue.
+#.  When Provision is ready ``Apple Home`` ask where to add light select a room and click ``Continue``.
+#. Now you can rename a light default is ``Matter Accessory`` name for example ``Matter Light`` and click ``Continue``.
+#. Click ``Continue``
+#. Click at next window ``View at Home`` and you can controll now Green led by Moving bar.
+
+Device is now provisioned and Light controll is possible by `Apple Home` and use with ``Automation``.
+
+Provisioning with chip-tool
+***************************
+
+Using ``chip-tool`` you need to know Thread network active data and Device QR-code payload.
+
+Manual pair code:
+`34970112332`
+
+QR Payload:
+`MT:6FCJ142C00KA0648G00`
+
+How to get Thread network Active data set:
+
+Open terminal to your linux which is running ``Open Thread Border router`` and call next command:
+
+.. code-block:: console
+
+    sudo ot-ctl dataset active -x
+    35060004001fffe00c0402a0f7f8051000112233445566778899aabbccddee00030e4f70656e54687265616444656d6f0410445f2b5ca6f2a93a55ce570a70efeecb000300001a02081111111122222222010212340708fd110022000000000e0800000003601c0000
+    Done
+
+Provision device and update accesslist:
+
+Here is example flow for provision device to with  ``node_id`` 1.
+
+1. Flash builded binary to device and press ``Reset`` button
+#. After reset you shuold see slowly blinking blue led which indicate that Device is advertisment Matter ready device for provision
+#. Start ``chip-tool`` in interactive mode by the following command:
+
+    .. code-block:: console
+
+        chip-tool interactive start
+
+#. Provision device by ``chip-tool`` and assign ``node_id`` 1 and QR code Payload``MT:6FCJ142C00KA0648G00`` by following command
+
+    .. code-block:: console
+
+        pairing code-thread 1 hex:35060004001fffe00c0402a0f7f8051000112233445566778899aabbccddee00030e4f70656e54687265616444656d6f0410445f2b5ca6f2a93a55ce570a70efeecb000300001a02081111111122222222010212340708fd110022000000000e0800000003601c0000 MT:6FCJ142C00KA0648G00 --bypass-attestation-verifier true
+
+#. Blue led start blinking faster when device ble connection is established and provision start to Thread and Matter Fabric.
+#. Wait That ``chip-tool`` finnish provisions.
+#. Update Light bulb's access list to include the light switch by the following command for ``node-id`` 1:
+
+   .. code-block:: console
+
+        accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [2], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}]' 1 0
+
+
+Light Bulb is ready for bind with light switch and control led by ``chip-tool``.
```

Comment:
Light Bulb is ready for bind with light switch and control led by ->
Light Bulb is ready to be bound with light switch and be controlled by